GCDWebServer.xcodeproj/project.pbxproj

```diff
@@ -861,13 +861,14 @@
 					};
 					CEE28CEE1AE0051F00F4023C = {
 						CreatedOnToolsVersion = 6.3;
-						DevelopmentTeam = 88W3E55T4B;
+						ProvisioningStyle = Manual;
```

Comment:
No, This issue only affects dynamic frameworks, not apps. Having Development team in a dynamic framework makes it unusable.